packages/trader/src/AppV2/Components/TradeParameters/Stake/stake.tsx

```diff
@@ -94,6 +96,22 @@ const Stake = observer(({ is_minimized }: TStakeProps) => {
         second_contract_payout,
     });
 
+    React.useEffect(() => {
+        if ((two_contracts_error || stake_error) && !is_minimized && !displayed_error.current) {
+            displayed_error.current = true;
+            addSnackbar({
+                message: <Localize i18n_default_text='Please adjust your stake.' />,
+                status: 'fail',
+                hasCloseButton: true,
+                style: { marginBottom: '48px' },
+            });
+        }
+    }, [two_contracts_error, stake_error, contract_type]);
+
+    React.useEffect(() => {
+        displayed_error.current = false;
+    }, [contract_type, symbol]);
```

Comment:
@henry-deriv contract_type is used in both useEffect, is it intentional ? if yes then why ?